MobileNav/SDL/ProxyManager.swift

```diff
@@ -114,6 +114,13 @@ class ProxyManager: NSObject {
         let streamingMediaConfig = SDLStreamingMediaConfiguration()
         streamingMediaConfig.carWindowRenderingType = getSDLRenderType(from: streamSettings.renderType)
 
+        streamingMediaConfig.supportedPortraitStreamingRange = SDLVideoStreamingRange(minimumResolution: SDLImageResolution(width: 300, height: 500), maximumResolution: SDLImageResolution(width: UInt16.max, height: UInt16.max))
+        streamingMediaConfig.supportedLandscapeStreamingRange = SDLVideoStreamingRange(minimumResolution: SDLImageResolution(width: 500, height: 300), maximumResolution: SDLImageResolution(width: UInt16.max, height: UInt16.max))
+        streamingMediaConfig.delegate = ProxyManager.sharedManager.self
```

Comment:
```suggestion
        streamingMediaConfig.delegate = self
```

Reply:
Changing to `self` gave me a compile error.

Reply:
Ah, I see, it's a class method